packages/browser-sdk/src/feedback/promptStorage.ts

```diff
@@ -16,7 +16,7 @@ export const checkPromptMessageCompleted = (
   userId: string,
   promptId: string,
 ) => {
-  const id = Cookies.get(`bucket-prompt-${userId}`);
+  const id = Cookies.get(`reflag-prompt-${userId}`);
```

Comment:
we use this to ensure that prompt messages are not shown twice. I think we'll need to check both cookies for now to avoid a situation where the "bucket" cookie is set, but the other is not